jx/JXML.js

```diff
@@ -54,3 +54,30 @@ JXML.deepMerge = function deepMerge(dst, src) {
 			dst[k] = v;
 	}
 }
+
+/**
+ * Same as deepMerge but returns data that actually changed.
```

Comment:
Which is by design: remove a parent removes the child. The 'diff' is actually the 'command' required to achieve the same attributes.